src/context.js

```diff
@@ -65,6 +71,28 @@ export class ReaderProvider extends Component {
     }))
   }
 
+  toggleFullscreen = () => {
+    const { fullscreen } = this.state
+    if (fullscreen) {
+      exitFullscreen()
+    } else {
+      requestFullscreen(document.querySelector('body'))
```

Comment:
Needs to request `fullscreen` for the viewer component not 'body',
use react ref.